### ISSUE TITLE
sabnzbd 1.2.0 - Issue with resuming paused queue

### ISSUE THREAD

**Issue by cdloh** [sabnzbd/sabnzbd#802]
I'm not sure if I'm doing something wrong or if it's a bug but thought I would report it anyway.

In my configuration I have two schedules setup.

1. 02:15 Daily Resume
2. 07:00 Daily Pause

Pretty simple. Just so that articles only download overnight.

The problem is that it doesn't appear to be unpausing it correctly. I have two options ticked that I think might be attributing to it.

In the Queue options I have 'Check before download' & 'Abort jobs that cannot be completed'. Not sure if that would make sab not want to resume the jobs as they are checking if they should be downloaded first?

I grabbed this information from the logs.

```
2017-01-30 02:15:00,000::INFO::[downloader:255] Resuming
2017-01-30 02:15:00,001::DEBUG::[__init__:583] PAUSED_ALL inactive
2017-01-30 07:00:00,003::INFO::[downloader:265] Pausing
```

As you can see it isn't doing anything at all between that time period not even attempting to connect to any servers.

Any ideas?

**Comment by Safihre** [comment]
Also reported by others: https://forums.sabnzbd.org/viewtopic.php?f=3&t=22264

We're not sure of the cause. Could you try this:
- Disable `Check before download`
- If it doesn't work: add an extra schedule task to restart SABnzbd at `02:00`.

**Comment by cdloh** [comment]
Phew. Great to hear it's not just me.

I've disable check before download and report back.

**Comment by Safihre** [comment]
@cdloh What OS are you on?

**Comment by cdloh** [comment]
Ubuntu 14.04 

Stock Python 2.7.6

**Comment by Safihre** [comment]
@cdloh 
Did it work to disable `Check before download`?
Could you try this version of SABnzbd? https://github.com/sabnzbd/sabnzbd/archive/bugfix/no_q_trylist.zip
Hoping this one will also work with the check turned on.

**Comment by cdloh** [comment]
@Safihre it has been working, however I there were days where it would go without hitting the bug so I was hesitant to jump to conclusions.

I'll get that version of sab installed in the next day or two and confirm if all is working correctly.

**Comment by Safihre** [comment]
@cdloh Could you install the new 1.2.1 and see if you still have any problems?

**Comment by Safihre** [comment]
@cdloh any more problems with the resume?

**Comment by Safihre** [comment]
Closing this due to lack of feedback.
In case the issue is not fixed in 1.2.2, re-open it 👍
